src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
    :connect                               "Connecter"
    :address                               "Adresse"
    :password                              "Mot de passe"
-   :login                                 "Connexion"
    :sign-in-to-status                     "S'inscrire à Status"
    :sign-in                               "Inscription"
    :wrong-password                        "Mot de passe incorrect"
+   :enter-password                        "Saisir le mot de passe"
 
    ;recover
-   :recover-from-passphrase               "Restaurer avec la phrase secrète"
-   :recover-explain                       "Veuillez saisir la phrase secrète correspondant à votre mot de passe pour restaurer votre accès"
    :passphrase                            "Phrase secrète"
    :recover                               "Restaurer"
-   :enter-valid-passphrase                "Veuillez saisir une phrase secrète"
-   :enter-valid-password                  "Veuillez saisir un mot de passe"
    :twelve-words-in-correct-order         "12 mots dans l'ordre"
 
    ;accounts
    :recover-access                        "Restaurer l'accès"
-   :add-account                           "Ajouter un compte"
    :create-new-account                    "Créer un nouveau compte"
 
    ;wallet-qr-code
    :done                                  "Terminé"
-   :main-wallet                           "Portefeuille principal"
 
    ;validation
    :invalid-phone                         "Numéro de téléphone invalide"
    :amount                                "Montant"
-   :not-enough-eth                        (str "Pas assez d'ETH sur le solde "
-                                               "({{balance}} ETH)")
+   
    ;transactions
    :confirm                               "Confirmer"
-   :confirm-transactions                  {:one   "Confirmer la transaction"
-                                           :other "Confirmer {{count}} transactions"
-                                           :zero  "Aucune transaction"}
-   :transactions-confirmed                {:one   "Transaction confirmée"
-                                           :other "{{count}} transactions confirmées"
-                                           :zero  "Aucune transaction confirmée"}
    :transaction                           "Transaction"
-   :unsigned-transactions                 "Transactions non signées"
-   :no-unsigned-transactions              "Aucune transaction non signée"
-   :enter-password-transactions           {:one   "Confirmez la transaction en saisissant votre mot de passe"
-                                           :other "Confirmez les transactions en saisissant votre mot de passe"}
+   :unsigned-transaction-expired          "La transaction non signée a expirée"
```

Comment:
> a expirée
should be "a expiré" ?

Reply:
I believe `expirée` is the correcte orthography as `La transaction` is a feminine noun

Reply:
Le participe passé ne s'accorde pas avec l'auxiliaire `avoir`